sql_upgrade.php

```diff
@@ -106,7 +106,8 @@
     let serverPaused = 0;
     // recursive long polling where ending is based
     // on global doPoll true or false.
-    async function serverStatus(version = '', start = 0) {
+    // added a forcePollOff parameter to avoid polling erroneously on indefinitely when updating from patch.sql
```

Comment:
minor typo
"or indefinitely?"

Reply:
will change 'erroneously' to 'from staying'